src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoMend.kt

```diff
@@ -90,6 +95,8 @@ internal object AutoMend : Module(
                     player.inventory.currentItem = xpSlot
                 }
                 if (autoThrow && player.heldItemMainhand.item === Items.EXPERIENCE_BOTTLE) {
+                    // This might trigger "too many packets"
+                    connection.sendPacket(CPacketPlayer.Rotation(player.rotationYaw, 90.0f, player.onGround))
```

Comment:
This will trigger for sure, just set a priority for the module and use PlayerPacketManager to do this

Reply:
The problem with PlayerPacketManager is that it only updates the rotation onUpdateWalkingPlayerEvent, which means we'll need to know before we throw to update the rotation.

I'm not sure of a good way to do that except to add a "stage" system to first rotate and then throw on the next tick.

Reply:
Do the rotation on pre phase and throw the bottle at post phase

Reply:
I'm implementing the velocity addition to the throw so that it doesn't end up throwing off a cliff. I'm going to do some rayTrace experiments to get it right to know if we will hit a block within pick up range.

Reply:
I decided against the velocity for now because my brain is not awake enough to figure out the right trig to use to guess the block we need to absorb the orbs perfectly, so it's the same "current" functionality other than throwing it at your feet instead of where you are looking.